packages/selleo-design-core/src/Sidebar/SidebarItem.tsx

```diff
@@ -0,0 +1,27 @@
+import { h } from "preact";
+import cx from "classnames";
+
+type Props = {
+  icon(): h.JSX.Element;
+  label: string;
+  active?: boolean;
+};
+
+export function SidebarItem({ icon, label, active }: Props) {
+  const IconComponent = icon;
+  const itemClasses = cx(
+    "flex items-center text-base leading-relaxed tracking-normal",
+    {
+      "text-neutral-600 dark:text-white": !active,
+      "text-brand-primary-500": active,
+    }
+  );
+  return (
+    <li class="m-0 mb-5 cursor-pointer list-none">
+      <div class={itemClasses}>
+        <IconComponent />
+        <span class="ml-[12px]">{label}</span>
```

Comment:
possible to round it to nearest variable?

Reply:
just let @michalgren know u did it cause the sidebar design is just taken from another app so it might be little off when it comes to consistency